### DISCUSSION TITLE
Screen not responding

### DISCUSSION THREAD

**Discussion by manmorlam** [driedpampas/realme-8-megaguide#34] [Q&A]
Hello, I have a problem following the guide. After Downgrade to RUI2 (Haadi's A.19 RUI2 Firmware) the phone rebooted and the boot wizard to select the language appears, but the screen is not responding. The volume and power buttons works, but I can't do anything on the screen.
I unchecked the opporeserve2 and cdt_engineering options and the process finished successfully. My phone is the European version (RMX3085).
What is happening? How can I fix it?
Thank you very much.

**Comment by lavafrai** [comment]
I have the same problem. Could you solve it?

**Reply by manmorlam** [reply]
You don't have to do anything after downgrading to RUI2. Skip directly to step 3 Unlocking the bootloader. Reboot the phone and run the console commands in the MTK Client folder.
I managed to complete the process.
Cheers.
